src/shieldpy/watertank.py

```diff
@@ -0,0 +1,21 @@
+import nfa
```

Comment:
i learn something new about python relative imports every day. i'm surprised this works, i'd have thought you need `from shieldpy import nfa`. haha anyway, below it doesn't say `nfa.NFA`, which seems weird to me.

Reply:
looks like it should've been nfa.NFA the build doesn't seem to complain about this but just noticed errors from my LSP

Reply:
I've changed it to the other type of import